examples/complete/README.md

```diff
@@ -1,4 +1,4 @@
-# Encryption example
+# Complete example with Encryption and CBR rules
```

Comment:
the rest of this readme is out of date and does not reflect what is in the module